dotCMS/src/main/webapp/html/portlet/ext/contentlet/field/edit_field.jsp

```diff
@@ -243,7 +243,7 @@
 
                     if (content) {
                         blockEditor.value = content;
-                        field.value = JSON.stringify(block.editor.getJSON());
+                        field.value = JSON.stringify(block?.editor.getJSON());
```

Comment:
Why `editor` might be falsey at this point? isn't that making the value `undefined` or `null`?